C/Checkpoints/Deps.toml

```diff
@@ -0,0 +1,24 @@
+[0]
+DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
+FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
+
+["0-0.1"]
+AWSCore = "4f1ea46c-232b-54a6-9b17-cc2d0f3e6598"
+AWSSDK = "0d499d91-6ae5-5d63-9313-12987b87d5ad"
+BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Mocking = "78c3b35d-d492-501b-9361-3d52fe80e533"
+
+["0-0.2.1"]
+AWSTools = "83bcdc74-1232-581c-948a-f29122bf8259"
```

Comment:
Looks like the history will require that AWSTools gets registered in General first.

Reply:
Should work after https://github.com/JuliaRegistries/General/pull/26003 is merged.